addons/account_check_printing/models/account_journal.py

```diff
@@ -102,3 +102,10 @@ def action_checks_to_print(self):
                 default_payment_method_line_id=payment_method_line.id,
             ),
         }
+
+    @api.model
+    def _get_reusable_payment_methods(self):
+        """ We are able to have multiple times Checks payment method in a journal """
+        res = super()._get_reusable_payment_methods()
+        res.append("check_printing")
```

Comment:
```suggestion
        res.add("check_printing")
```
See the change above